src/it/scala/org/broadinstitute/dsde/rawls/openam/RawlsOpenAmClientSpec.scala

```diff
@@ -0,0 +1,25 @@
+package org.broadinstitute.dsde.rawls.openam
+
+import java.util.concurrent.TimeUnit
+
+import org.broadinstitute.dsde.rawls.IntegrationTestConfig
+import org.broadinstitute.dsde.vault.common.openam.OpenAMResponse.AuthenticateResponse
+import org.scalatest.{Matchers, WordSpecLike}
+
+import scala.concurrent.Await
+import scala.concurrent.duration.FiniteDuration
+
+class RawlsOpenAmClientSpec extends WordSpecLike with Matchers with IntegrationTestConfig {
+  val timeoutDuration = new FiniteDuration(5, TimeUnit.SECONDS)
+
+  "RawlsOpenAmClient" should {
+    "return a valid token id from authenticate" in {
+      val result = Await.result(RawlsOpenAmClient.authenticate, timeoutDuration)
+      result shouldNot be (None)
+      result.asInstanceOf[AuthenticateResponse].tokenId shouldNot be (null)
+      result.asInstanceOf[AuthenticateResponse].successUrl shouldNot be (null)
+    }
+
+    // TODO test other stuff
```

Comment:
I put in this TODO, I think I was thinking of testing the other methods in there (e.g. ```lookupUserInfo```), not sure if it's necessary though